loom/database/clients/mongodb_clients.py

```diff
@@ -87,9 +114,18 @@ def links(self) -> AgnosticCollection:
     def aliases(self) -> AgnosticCollection:
         return self.database.aliases
 
+    async def authenticate(self, username, password):
```

Comment:
While we do not use it, it does allow for post-initialization authentication in the event that you want to adjust authentication after creating a client or interface. Still want me to remove it?